pyproject.toml

```diff
@@ -36,10 +36,10 @@ dependencies = [
     "rioxarray~=0.13.4",
     "s3fs~=2024.9.0",
     "xarray~=2024.9.0",
-    "geojson-pydantic>=1.1.1",
+    "geojson-pydantic>=2.0.0,<3.0",
     "python-dateutil>=2.9.0.post0",
     "httpx>=0.27.2",
-    "pillow>=11.0.0",
+    "pillow==11.2.1",
```

Comment:
I was getting a pillow build error, which may be related to the release yesterday? Looking at this issue: https://github.com/python-pillow/Pillow/issues/906, the [titiler-cmr build failures are similar](https://github.com/developmentseed/titiler-cmr/actions/runs/16030522087/job/45229930367)

Reply:
it looks like this issue was resolved by `uploaded manylinux2014 wheels to PyPI`